scripts/generate_ledgers_tar_from_config.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+# verify-pacakged-fork-config.sh inlines this file- keep them in sync
```

Comment:
This file probably needs to be removed, we should use scripts that are executed by CI (`scripts/hardfork/create_runtime_config.sh` in this case)